physionet-django/physionet/test_gcs.py

```diff
@@ -0,0 +1,207 @@
+import os
+from unittest import skipIf
+
+from django.test import TestCase
+from django.test import override_settings
+from google.api_core.client_options import ClientOptions
+from google.auth.credentials import AnonymousCredentials
+from google.cloud.exceptions import NotFound
+from google.cloud.storage import Client, Bucket, Blob
+
+from physionet.gcs import GCSObject, GCSObjectException
+from physionet.settings.base import StorageTypes
+
+
+SKIP__GCS_INTEGRATION = os.environ.get('TEST_GCS_INTEGRATION') == 'false'
```

Comment:
I'd use decouple here like so:

```py
from decouple import config

SKIP__GCS_INTEGRATION = not config('TEST_GCS_INTEGRATION', default=False, cast=bool)
```

By the way, is there a reason for the double underscore here?